TheBTeam.BLL/Services/Edits.cs

```diff
@@ -0,0 +1,297 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TheBTeam.BLL.Model;
+using TheBTeam.BLL.Validators;
+
+namespace TheBTeam.BLL.Services
+{
+    public class Edits
+    {
+        const int MinAddressLength = 3;
+        const int MinNameLength = 2;
+        private const int MinPhoneNumberLength = 12;
+        const int MinAge = 13;
+        const int MaxAge = 99;
+        public static void EditUser(User user)
+        {
+            var tempuser = new User(user.FirstName, user.LastName, user.Gender, user.Age, user.Email, user.Phone, user.Address, user.Company, user.Currency, user.Balance);
+            string options = "1)Edit surname\n2)Edit age\n3)Edit Company\n4)Edit gender\n5)Edit phone\n6)Edit balance\n7)Edit currency\n8)Finish";
+            bool exit = false;
+            while (exit == false)
+            {
+                Console.WriteLine(options, "\n");
+                string choice = Console.ReadLine();
+                if (choice == "1")
```

Comment:
it is ok, bur maby switch case would be more readable?

Reply:
Ur right, done as per convention